tests/lib/schema_validation_test.py

```diff
@@ -231,6 +231,73 @@ def test_over_maximum_total_vfs_is_invalid(
         with pytest.raises(js.ValidationError, match=str(invalid_values)):
             libnmstate.validator.validate(default_data)
 
+    @pytest.mark.parametrize("invalid_values", [-50, -1])
```

Comment:
`invalid_values` >> `invalid_value`
I would actually prefer it named per what it represent: `vf_if`.

The test name can also be generalized to represent the invalid ids: `test_invalid_vf_ids`

Reply:
Sure, that seems better.

Reply:
Why not `vf_if`? Note that currently it is also plural, but the value is singular.
What about the test name?